osu.Game/Skinning/Components/DefaultTextSkinComponent.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+
+namespace osu.Game.Skinning.Components
+{
+    /// <summary>
+    /// Skin element that contains text and have ability to control its font.
+    /// </summary>
+    public abstract partial class DefaultTextSkinComponent : Container, ISkinnableDrawable
```

Comment:
What's the `Default` for in the name of this class? I don't quite get it.

Reply:
Wanted to distinguish it from components based on "legacy" font, but I agree that it doesn't sound well.

Reply:
I've renamed this.